pkg/kube/controllers/extendedjob/errand_reconciler.go

```diff
@@ -70,6 +72,11 @@ func (r *ErrandReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 		return result, err
 	}
 
+	if meltdown.InWindow(time.Now(), r.config.MeltdownDuration, eJob.ObjectMeta.Annotations) {
+		ctxlog.WithEvent(eJob, "Meltdown").Debugf(ctx, "Resource '%s' is in meltdown, delaying reconciles for %s", eJob.Name, r.config.MeltdownDuration)
+		return reconcile.Result{Requeue: true, RequeueAfter: r.config.MeltdownRequeueAfter}, nil
```

Comment:
We could only set `RequeueAfter > 0` to requeue the request: https://github.com/cloudfoundry-incubator/cf-operator-vendor/blob/e337a0cdf3c31f7053be70212b3059d078ed8fa1/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go#L221